ui/nuxeo-video/nuxeo-video-conversions.js

```diff
@@ -0,0 +1,84 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
+import { FormatBehavior } from '../nuxeo-format-behavior.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import '../nuxeo-icons.js';
+import '../widgets/nuxeo-tooltip.js';
+
+{
+  /**
+  `nuxeo-video-conversions`
+  @group Nuxeo UI
+  @element nuxeo-video-conversions
+  */
+  class VideoConversions extends mixinBehaviors([I18nBehavior, FormatBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          a,
+          a:active,
+          a:visited,
+          a:focus {
+            @apply --nuxeo-link;
+          }
+          a:hover {
+            @apply --nuxeo-link-hover;
+          }
+        </style>
+        <h3>[[label]]</h3>
```

Comment:
Looking at the JSDoc for this property "optional", I would suggest to perform a small improvement and wrap this `<h3>` inside a `dom-if`. Because, as is, if we don't pass any value for the label, we'll render an empty `h3` on our DOM.

Reply:
to be honest i was thinking of not having the h3 as it is not part of the core functionality...

Reply:
followed the suggestion and wrapped it into a dom-if